### ISSUE TITLE
[Bug] can't build 

### ISSUE THREAD

**Issue by YYGod0120** [farm-fe/farm#703]
### Steps to reproduce

`pnpm farm build`

### What is expected?

can build the project

### What is actually happening?

```
[ Farm ] Using config file at E:\cqupt-helper-backage\farm.config.ts

 Build failed due to errors:

 Can not resolve `zlib` from node_modules/.pnpm/pngjs@5.0.0/node_modules/pngjs/lib/packer.js.
Original error: None.

Potential Causes:
1.The file that `zlib` points to does not exist.
2.Install it first if `zlib` is an dependency from node_modules, if you are using pnpm refer to [https://pnpm.io/faq#pnpm-does-not-work-with-your-project-here] for solutions.
3. If `zlib` is a alias, make sure your alias config is correct.

Can not resolve `util` from node_modules/.pnpm/pngjs@5.0.0/node_modules/pngjs/lib/png.js.
Original error: None.

Potential Causes:
1.The file that `util` points to does not exist.
2.Install it first if `util` is an dependency from node_modules, if you are using pnpm refer to [https://pnpm.io/faq#pnpm-does-not-work-with-your-project-here] for solutions.
3. If `util` is a alias, make sure your alias config is correct.

Can not resolve `zlib` from node_modules/.pnpm/pngjs@5.0.0/node_modules/pngjs/lib/packer-sync.js.
Original error: None.

Potential Causes:
1.The file that `zlib` points to does not exist.
2.Install it first if `zlib` is an dependency from node_modules, if you are using pnpm refer to [https://pnpm.io/faq#pnpm-does-not-work-with-your-project-here] for solutions.
3. If `zlib` is a alias, make sure your alias config is correct.

Can not resolve `stream` from node_modules/.pnpm/pngjs@5.0.0/node_modules/pngjs/lib/png.js.
Original error: None.

Potential Causes:
1.The file that `stream` points to does not exist.
2.Install it first if `stream` is an dependency from node_modules, if you are using pnpm refer to [https://pnpm.io/faq#pnpm-does-not-work-with-your-project-here] for solutions.
3. If `stream` is a alias, make sure your alias config is correct.

Can not resolve `zlib` from node_modules/.pnpm/pngjs@5.0.0/node_modules/pngjs/lib/parser-sync.js.
Original error: None.

Potential Causes:
1.The file that `zlib` points to does not exist.
2.Install it first if `zlib` is an dependency from node_modules, if you are using pnpm refer to [https://pnpm.io/faq#pnpm-does-not-work-with-your-project-here] for solutions.
3. If `zlib` is a alias, make sure your alias config is correct.

Can not resolve `util` from node_modules/.pnpm/pngjs@5.0.0/node_modules/pngjs/lib/parser-async.js.
Original error: None.

Potential Causes:
1.The file that `util` points to does not exist.
2.Install it first if `util` is an dependency from node_modules, if you are using pnpm refer to [https://pnpm.io/faq#pnpm-does-not-work-with-your-project-here] for solutions.
3. If `util` is a alias, make sure your alias config is correct.

Can not resolve `zlib` from node_modules/.pnpm/pngjs@5.0.0/node_modules/pngjs/lib/parser-async.js.
Original error: None.

Potential Causes:
1.The file that `zlib` points to does not exist.
2.Install it first if `zlib` is an dependency from node_modules, if you are using pnpm refer to [https://pnpm.io/faq#pnpm-does-not-work-with-your-project-here] for solutions.
3. If `zlib` is a alias, make sure your alias config is correct.

Can not resolve `util` from node_modules/.pnpm/pngjs@5.0.0/node_modules/pngjs/lib/packer-async.js.
Original error: None.

Potential Causes:
1.The file that `util` points to does not exist.
2.Install it first if `util` is an dependency from node_modules, if you are using pnpm refer to [https://pnpm.io/faq#pnpm-does-not-work-with-your-project-here] for solutions.
3. If `util` is a alias, make sure your alias config is correct.

Can not resolve `stream` from node_modules/.pnpm/pngjs@5.0.0/node_modules/pngjs/lib/packer-async.js.
Original error: None.

Potential Causes:
1.The file that `stream` points to does not exist.
2.Install it first if `stream` is an dependency from node_modules, if you are using pnpm refer to [https://pnpm.io/faq#pnpm-does-not-work-with-your-project-here] for solutions.
3. If `stream` is a alias, make sure your alias config is correct.

Can not resolve `stream` from node_modules/.pnpm/pngjs@5.0.0/node_modules/pngjs/lib/chunkstream.js.
Original error: None.

Potential Causes:
1.The file that `stream` points to does not exist.
2.Install it first if `stream` is an dependency from node_modules, if you are using pnpm refer to [https://pnpm.io/faq#pnpm-does-not-work-with-your-project-here] for solutions.
3. If `stream` is a alias, make sure your alias config is correct.

Can not resolve `util` from node_modules/.pnpm/pngjs@5.0.0/node_modules/pngjs/lib/chunkstream.js.
Original error: None.

Potential Causes:
1.The file that `util` points to does not exist.
2.Install it first if `util` is an dependency from node_modules, if you are using pnpm refer to [https://pnpm.io/faq#pnpm-does-not-work-with-your-project-here] for solutions.
3. If `util` is a alias, make sure your alias config is correct.

Can not resolve `util` from node_modules/.pnpm/pngjs@5.0.0/node_modules/pngjs/lib/filter-parse-async.js.
Original error: None.

Potential Causes:
1.The file that `util` points to does not exist.
2.Install it first if `util` is an dependency from node_modules, if you are using pnpm refer to [https://pnpm.io/faq#pnpm-does-not-work-with-your-project-here] for solutions.
3. If `util` is a alias, make sure your alias config is correct.

Can not resolve `assert` from node_modules/.pnpm/pngjs@5.0.0/node_modules/pngjs/lib/sync-inflate.js.
Original error: None.

Potential Causes:
1.The file that `assert` points to does not exist.
2.Install it first if `assert` is an dependency from node_modules, if you are using pnpm refer to [https://pnpm.io/faq#pnpm-does-not-work-with-your-project-here] for solutions.
3. If `assert` is a alias, make sure your alias config is correct.

Can not resolve `util` from node_modules/.pnpm/pngjs@5.0.0/node_modules/pngjs/lib/sync-inflate.js.
Original error: None.

Potential Causes:
1.The file that `util` points to does not exist.
2.Install it first if `util` is an dependency from node_modules, if you are using pnpm refer to [https://pnpm.io/faq#pnpm-does-not-work-with-your-project-here] for solutions.
3. If `util` is a alias, make sure your alias config is correct.

Can not resolve `zlib` from node_modules/.pnpm/pngjs@5.0.0/node_modules/pngjs/lib/sync-inflate.js.
Original error: None.

Potential Causes:
1.The file that `zlib` points to does not exist.
2.Install it first if `zlib` is an dependency from node_modules, if you are using pnpm refer to [https://pnpm.io/faq#pnpm-does-not-work-with-your-project-here] for solutions.
3. If `zlib` is a alias, make sure your alias config is correct.
```

### System Info

```Shell
System:
    OS: Windows 11 10.0.22621
    CPU: (16) x64 AMD Ryzen 7 5800H with Radeon Graphics
    Memory: 1.41 GB / 15.40 GB
  Binaries:
    Node: 18.17.1 - D:\node.EXE
    Yarn: 1.22.19 - D:\node_global\yarn.CMD
    npm: 9.7.2 - D:\npm.CMD
    pnpm: 8.7.5 - D:\node_global\pnpm.CMD
  Browsers:
    Edge: Chromium (118.0.2088.76)
    Internet Explorer: 11.0.22621.1
  npmPackages:
    @farmfe/core: ^0.13.16 => 0.13.16
```

### Any additional comments?

An error occurred when I migrated a project from vite to farm.
I want to try the speed of farm

**Comment by wre232114** [comment]
I'll take a look

**Comment by wre232114** [comment]
fixed in `@farmfe/core@0.13.20`
